beater/journalbeat.go

```diff
@@ -253,6 +265,44 @@ func (jb *Journalbeat) convertMicrosecondsEpochToISO8601(microsecondsEpoch int64
 // Run is the main event loop: read from journald and pass it to Publish
 func (jb *Journalbeat) Run(b *beat.Beat) error {
 	logp.Info("Journalbeat is running!")
+	if jb.config.MetricsEnabled {
+		fmt.Println("Metrics are enabled", jb.config.WavefrontCollector)
+		addr, err := net.ResolveTCPAddr("tcp", jb.config.WavefrontCollector)
+		if jb.config.WavefrontCollector != "" && err == nil {
```

Comment:
My hope was resolve TCP addr will throw an error if that was the case but it does not. 
Error handling code in go is little painful so combining it since semantically there is no difference.